apps/ubuntu_bootstrap/lib/pages/storage/passphrase/passphrase_widgets.dart

```diff
@@ -6,6 +6,8 @@ import 'package:ubuntu_bootstrap/pages/storage/passphrase/passphrase_model.dart'
 import 'package:ubuntu_widgets/ubuntu_widgets.dart';
 import 'package:yaru/icons.dart';
 
+// TODO: this will need to be conditionally hooked up to the /storage/v2/calculate_entropy
+//       endpoint in Subiquity once it is available
```

Comment:
I guess that also means we'll do the `subiquity_client` changes to support the new endpoint in a separate PR?

Reply:
Yes, I shared a quick update in the team channel in Mattermost: the snapd API isn't ready to integrate with yet so they would prefer we don't hook anything up at all (even the stub being provided by Subiquity).